_contentTemplates/editor/general.md

```diff
@@ -11,3 +11,30 @@ This section applies only to Blazor **Server** apps. Blazor **WebAssembly** apps
 Blazor **Server** apps use the **SignalR WebSocket** to send the Editor `Value` from the browser to the server .NET runtime and vice-versa. The default SignalR maximum message size is **32 KB**. To work with larger content (especially paste images as Base64 strings), [increase the max WebSocket message size for the Blazor application]({%slug common-kb-increase-signalr-max-message-size%}).
 
 #end
+#prosemirror-schema-prerequisites
+## Prerequisites
+
+Customization of the ProseMirror schema requires familiarity with:
+
+* JavaScript - the ProseMirror library is JavaScript-based and working with the schema is performed through JavaScript.
+* [ProseMirror Schema](https://prosemirror.net/docs/guide/#schema) - the schema structure and its children ([nodes](https://prosemirror.net/docs/ref/#model.NodeType), [marks](https://prosemirror.net/docs/ref/#model.MarkType)).
+#end
+
+#prosemirror-schema-general-info
+The Editor accepts a custom ProseMirror schema through the `Schema` parameter(`string`). Use this parameter to provide the name of your JS function that returns your custom schema - an instance of the [ProseMirror `Schema` class](https://prosemirror.net/docs/ref/#model.Schema).
+
+This JS function:
+* Must be declared in the global scope (the `window` object).
+* Must return an instance of the [ProseMirror `Schema` class](https://prosemirror.net/docs/ref/#model.Schema)(the updated schema). You can access this class from the `ProseMirror` object of the event arguments.
+* Provides an `arguments` object with the following properties:
```

Comment:
"Provides" is unclear. A function either accepts (expects) arguments of specific type (and structure), or returns a value of specific type. Which one is it?